### PR TITLE
Making base docker-compose run simple signer; cleaning up .env.defaults

### PR DESCRIPTION
**Pull request by mattschoch**
Helps streamline local setup.
You can run `make setup`, `make docker/stack/build`, then `make docker/stack/up` and get the full stack live, can go to devtool (now included in compose file) and start using it.

### CODE REVIEW

File: apps/armory/.env.default [hunk 1]
```diff
@@ -2,13 +2,15 @@ NODE_ENV=development
...
-APP_UID=local-dev-armory-instance-1
-
 # OPTIONAL: Sets the admin API key instead of generating a new one during the
 # provision.
 #
-# Plain text API key: 2cfa9d09a28f1de9108d18c38f5d5304e6708744c7d7194cbc754aef3455edc7e9270e2f28f052622257
-ADMIN_API_KEY=7d4064073efcfef92d85f274c70e82f980128168e9fb171a613dc7fdaf446b5d
+# Key should be hashed, like this: `echo -n "my-api-key" | openssl dgst -sha256 | awk '{print $2}'`
+# Plain text API key: armory-admin-api-key
+ADMIN_API_KEY=171c50ec62122b8c08362dcf9dce9b016ed615cfc7b90d4bc3fe5b223d967fb2
```
Comment (wcalderipe) [inline apps/armory/.env.default:10]: Nice, thanks for that.

File: README.md [hunk 2]
```diff
@@ -42,12 +42,10 @@ make docker/up
 make docker/stop
 ```
 
-Alternatively, you can run the entire stack in Docker containers. This is
-useful when using the MPC as a signing protocol in the Policy Engine.
+Alternatively, you can run the entire development stack in Docker containers.
```
Comment (wcalderipe) [inline README.md:45]: Can we add the command `docker/stack/up` so the reader now how to do it?

**Review by wcalderipe — APPROVED** [review]
All good. Make sure you run `make format` before merge.

File: README.md [hunk 3]
```diff
@@ -42,12 +42,10 @@ make docker/up
...
 
 > [!IMPORTANT]
-> You also need a locally running TSM cluster, which is not included in this
-> repository.
+> This builds a local dev image that mounts your filesystem into the container. Only db schema, dependency, or config changes require re-building. Do not use this build for production.
 
 ```bash
 # Build the application's image.
```
Comment (wcalderipe) [inline README.md:51]: A low hanging fruit, please add a requirement section somewhere in the readme with the following items

- Node version
- Docker
- OPA CLI (https://www.openpolicyagent.org/docs/latest/#running-opa)

File: README.md (hunk 2 above)
Comment (mattschoch) [reply README.md:45]: it's there, right below this

File: README.md (hunk 3 above)
Comment (mattschoch) [reply README.md:59]: Got it

**Pull request merged**